backend/program/symlink.py

```diff
@@ -80,7 +70,7 @@ def validate(self):
     def create_initial_folders(self):
         """Create the initial library folders."""
         try:
-            self.library_path = self.settings.host_path.parent / "library"
+            self.library_path = self.settings.container_path / "library"

```

Comment:
corrected this here since its what it should be set to

Reply:
renamed in #211